src/android/com/fivestars/mtab/plugin/RestartAppPlugin.java

```diff
@@ -39,6 +45,11 @@ public boolean execute(String action, JSONArray args, final CallbackContext call
     }
 
     private boolean restartApp() {
+        // use a delay of 2000 ms and exit the app by default
+        return restartApp(2000L, true);

```

Comment:
A constant for 2000L would be nice. Make updating this delay easier in the future.